ocaml/xapi/test_clustering.ml

```diff
@@ -207,11 +207,50 @@ let test_assert_cluster_host_is_enabled_for_matching_sms =
   ; "test_assert_cluster_host_is_enabled_for_matching_sms_succeeds_if_no_cluster_host_exists_and_clustering_is_not_needed" >:: test_assert_cluster_host_is_enabled_for_matching_sms_succeeds_if_no_cluster_host_exists_and_clustering_is_not_needed
   ]
 
+let test_assert_pif_prerequisites () =
+  let __context = Test_common.make_test_database () in
+  let network = Test_common.make_network ~__context () in
+  let localhost = Helpers.get_localhost ~__context in
+  let (_cluster, _cluster_host) = Test_common.make_cluster_and_cluster_host ~__context ~network ~host:localhost () in
```

Comment:
The test doesn't seem to use these values, does it rely on some side effect of this, or can we safely delete this line?

Reply:
True, they are not used at the moment. However, this is just an old test that I've moved and then had to make some changes to make it compile. Maybe these are created to make the test more realistic 🤔

Reply:
Ok lets keep them.